src/protocol/RemotingCommand.cpp

```diff
@@ -302,6 +302,10 @@ void RemotingCommand::setMsgBody(const string& body) {
   m_msgBody = body;
 }
 
+void RemotingCommand::moveMsgBody(std::string&& body) {
```

Comment:
Why not provide an overload method of SetBody, accepting a r-value reference to string

Reply:
> Why not provide an overloading method of SetBody, accepting a r-value reference to string

because it is more easy to understand the "move" of the function，if overload method of SetBody,I think developer may use reference method, but not use r-value reference method~~